src/components/PeopleFilters.tsx

```diff
@@ -33,63 +94,68 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
+            <Link
               data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
+              className={`button mr-1 ${centuries.includes('16') ? 'is-info' : ''}`}
+              to="#"
+              onClick={e => handleCenturyClick(e, '16')}
             >
               16
-            </a>
+            </Link>
 
-            <a
+            <Link
               data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
+              className={`button mr-1 ${centuries.includes('17') ? 'is-info' : ''}`}
+              to="#"
+              onClick={e => handleCenturyClick(e, '17')}
             >
               17
-            </a>
+            </Link>
 
-            <a
+            <Link
               data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
+              className={`button mr-1 ${centuries.includes('18') ? 'is-info' : ''}`}
+              to="#"
+              onClick={e => handleCenturyClick(e, '18')}
             >
               18
-            </a>
+            </Link>
 
-            <a
+            <Link
               data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
+              className={`button mr-1 ${centuries.includes('19') ? 'is-info' : ''}`}
+              to="#"
+              onClick={e => handleCenturyClick(e, '19')}
```

Comment:
can you think of a way to simplify this handle here to only receive the event? (hint: ids)